config.mk.tmp

```diff
@@ -0,0 +1,138 @@
+#-------------------------------------------------------------------------------
```

Comment:
I didn't mean a separate repo. There should be a option in config.mk called USE_MKLDNN and all the other flags should be added in Makefile if you see USE_MKLDNN=1